tensorflow_datasets/structured/somerville_happiness_test.py

```diff
@@ -0,0 +1,20 @@
+"""Test for Somerville Happiness Survey Data Set"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.structured import somerville_happiness
+
+
+class SomervilleHappinessTest(testing.DatasetBuilderTestCase):
```

Comment:
Did you run the tests, are they passing? Can you please add the output as a comment to the PR